partner_vat_unique/models/res_partner.py

```diff
@@ -14,14 +13,17 @@ class ResPartner(models.Model):
 
     @api.constrains("vat", "parent_id")
     def _check_vat_unique(self):
+        if (
+            not self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("partner_vat_unique.partner_vat_unique", default=False)
```

Comment:
IMO default could be `True`. I guess that whoever installs this module expects the constraint to apply.

At least the configuration must be explained in the readme.